config/core/build-configs.yaml

```diff
@@ -251,6 +251,29 @@ fragments:
       - 'CONFIG_MMC_SDHCI=y'
       - 'CONFIG_MMC_SDHCI_PCI=y'
       - 'CONFIG_MMC_SDHCI_ACPI=y'
+      - 'CONFIG_CHROME_PLATFORMS=y'
+      - 'CONFIG_CHROMEOS_LAPTOP=y'
+      - 'CONFIG_CHROMEOS_TBMC=y'
+      - 'CONFIG_DM_CRYPT=y'
+      - 'CONFIG_DM_THIN_PROVISIONING=y'
+      - 'CONFIG_DM_INIT=y'
+      - 'CONFIG_DM_VERITY=y'
```

Comment:
Just to make it clear these DM options are only useful to boot chromeos userspace (so more suitable for the chromeos.kernelci.org) instance. Not sure if @gctucker wants these options here.

Reply:
They are dropped now. marking as resolved.